d2.config.js

```diff
@@ -0,0 +1,9 @@
+const config = {
+    type: 'app',
```

Comment:
This needs a `title` property, otherwise what appears in the Headerbar and browser tab is "menu-management-app"

The name should also be overridden here as `menu-management`.  Finally, this needs to be flagged as a `core-app`

See https://github.com/dhis2/import-export-app/blob/master/d2.config.js#L3-L5